### PR TITLE
Bugfix/delete not cached file

### PR DESCRIPTION
**Pull request by mateussiqueira**
Remove unnecessary method createFile
Add Test to check if return null if file is not cached
Move const values to the setup

### CODE REVIEW

**Review by martijn00 — APPROVED** [review]

File: flutter_cache_manager/lib/src/cache_store.dart [hunk 1]
```diff
@@ -181,8 +182,9 @@ class CacheStore {
     if (_futureCache.containsKey(cacheObject.key)) {
       _futureCache.remove(cacheObject.key);
     }
-    final file = await fileSystem.createFile(cacheObject.relativePath);
-    if (await file.exists()) {
+    final file = io.File(cacheObject.relativePath);
```
Comment (jungho87) [inline flutter_cache_manager/lib/src/cache_store.dart:185]: @martijn00 
Files will not be deleted after this change. Previously, the location of the directory was set within the `fileSystem`, so can get file information by `relativePath`. But now, cannot get file information by `relativePath` because using `io.File`
